src/ui/components/component.js

```diff
@@ -22,7 +22,13 @@ export default class Component {
     this.moduleId = null;
 
     /**
-     * Unique name of this component instance
+     * UUID for the component.
+     * @type {number}
+     */
+    this.uuid = systemConfig.uuid;
+
+    /**
+     * Name of this component instance
      * Used to distinguish between other components of the same type
```

Comment:
I believe these changes would make this line outdated since name is no longer used to differentiate components of the same type